app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @new_comment = Comment.new
+  end
+
+  def create
+    # @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @new_comment = @post.comments.new(comment_params)
+    @new_comment.author_id = @post.id
+    @new_comment.post_id = params[:post_id]
+    @new_comment.update_comments_counter
+
+    if @new_comment.save
+      redirect_to user_post_path(@post.id, Post.find(params[:post_id]))
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] Creating a new comment doesn't work as expected, kindly fix it so that a user should create a new comment for each post or a random post.

![image](https://user-images.githubusercontent.com/55635977/160657813-f79fd0cf-fb6c-4b3c-a2e1-469a2b9c6b22.png)

![image](https://user-images.githubusercontent.com/55635977/160657702-e1380e27-2011-4863-ade9-cba8763663c0.png)